### ISSUE TITLE
Add endcomma: multiline option

### ISSUE THREAD

**Issue by InfernoZeus** [prettydiff/prettydiff#335]
Thoughts on adding an option to only enforce trailing commas when the array/object occurs over multilines? In my use-case, I tend to use trailing commas, so that new additions don't affect the previous line when doing a diff. With single line arrays/objects, such as imports,
```
import {Router, Route, browserHistory, IndexRoute,} from 'react-router'
```
adding a new import will obviously affect that line already. In this situation, the trailing comma doesn't add much.

**Comment by prettydiff** [comment]
Have you tried using the `endcomma` option?  http://prettydiff.com/documentation.xhtml#endcomma

I can see, though, that this option is missing from the diff mode options on the web tool.  I will add it.  If you are using Pretty Diff in a web browser try passing code through the *beautify* mode with option endcomma and let me know if that gets you closer to what you need.

**Comment by InfernoZeus** [comment]
Sorry if I wasn't clear. That would be perfect, thanks!

**Comment by prettydiff** [comment]
This feature is now available in 2.1.1 branch and will merge to master soon.  The `multiline` value is not applied by the parser as the application doesn't know if an object is multiline or not yet, so it is really only a vainly applied at the latest stages of beautification.
